pyiron_base/job/generic.py

```diff
@@ -1232,6 +1247,10 @@ def from_hdf(self, hdf=None, group_name=None):
         if "import_directory" in self._hdf5.list_nodes():
             self._import_directory = self._hdf5["import_directory"]
         self._server.from_hdf(self._hdf5)
+        try:
+            self._meta.from_hdf(self._hdf5)
+        except ValueError:
+            pass # no meta data written for this job previously
```

Comment:
Can this be more elegant by checking if the corresponding group exists in the HDF5 file